Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -106,12 +106,32 @@ extension SettingsAccountViewController: UITableViewDelegate {
   }
 
   func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
-    return 0.1
+    let (userHasPassword, _) = self.viewModel.outputs.userHasPasswordAndEmail
+    guard section == SettingsAccountSectionType.createPassword.rawValue, !userHasPassword else {
+      return 0.1
+    }
+    return Styles.grid(10)
```

Comment:
Looks like this breaks Dynamic Type support. If we set this directly we won't allow the footer to resize itself. I believe a better approach would be to do the following:

```suggestion
    return UITableView.automaticDimension
```

Which will allow the height to be dynamic..and in addition to that we also should set `estimatedSectionFooterHeight` in order to give it at least a bit of information. Isabel previously suggested this is a `static var` on the footer class itself so we could do:

```
final class SettingsGroupedFooterView: UITableViewHeaderFooterView {
  static var defaultHeight: CGFloat = 44
  ...
}
```